packages/dart/noports_core/lib/src/sshnp/util/sshnpd_channel/sshnpd_channel.dart

```diff
@@ -325,7 +372,8 @@ abstract class SshnpdChannel with AsyncInitialization, AtClientBindings {
       bool showHiddenKeys = false}) async {
     var builder = ScanVerbBuilder()
       ..sharedWith = sharedWith
-      ..sharedBy = sharedBy
+      ..sharedBy =
+          sharedBy // for some reason, if this is null it only returns cached keys
```

Comment:
Noted a behavioral bug, which explains why I went an round-about way to implement listDevices without -t